app/MMB/src/scenes/travelDocument/menuItem/PassengerMenuItem.js

```diff
@@ -23,6 +23,15 @@ class PassengerMenuItem extends React.Component<Props> {
     this.props.navigation.navigate({
       routeName: 'TravelDocumentModalScreen',
       key: 'key-TravelDocumentModalScreen',
+      params: {
+        fullName: `${idx(this.props.data, _ => _.title) || ''}. ${idx(
```

Comment:
This is pedantic, but I think whether there should be a full stop ( . ) after the title should be decided by the locale currently used: https://english.stackexchange.com/questions/374899/which-is-correct-dr-or-dr

Reply:
We could maybe pass the entre title through translations 🤔

Reply:
Maybe. Who is the translations expert here?

Reply:
I mean, it makes sense, now we just show what is passed through the graphql. But, if I put the existing app on Spanish, it shows me `Sr.` which is the spanish translation for `Mr.`